app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-schemas.js

```diff
@@ -60,9 +59,9 @@
     schema.event = eventAttribute
     // exceptions should be inserted without checking for sub parameters
     // exceptions should only come from code, not directly from data attributes
-    // e.g. link_path_parts is generated by JS and contains potentially non-unique
+    // e.g. an object generated by JS and contains potentially non-unique
     // sub parameter names i.e. '1', '2'
-    var exceptions = ['link_path_parts']
+    var exceptions = []
```

Comment:
While this might be useful at some point, unless we're about to need it or can think of a good reason to keep it, we should probably get rid of it entirely. What do you think?